apps/examples/dotnet-api/Features.cs

```diff
@@ -0,0 +1,160 @@
+using FeatureBoard.DotnetSdk.Attributes;
+using FeatureBoard.DotnetSdk.Models;
+using System.Runtime.Serialization;
+
+namespace FeatureBoardSdk.Examples.DotnetApi;
+
+// Features
+public class Features : IFeatures
```

Comment:
Prob should use a demo project rather than the FB service itself, might be easier for people to understand

Reply:
Agreed but I planed to fix this but need a prod instance to do this .

Reply:
Changed to sassy icons output